tests/BenchmarkDotNet.IntegrationTests/BuildTimeoutTests.cs

```diff
@@ -31,8 +31,8 @@ public void WhenBuildTakesMoreTimeThanTheTimeoutTheBuildIsCancelled()
                 .AddJob(Job.Dry
                     .WithRuntime(NativeAotRuntime.Net70)
```

Comment:
I just found a bug in my part.

```suggestion
                    .WithRuntime(NativeAotRuntime.Net80)
```